provider.json

```diff
@@ -1,39 +1,54 @@
 {
   "acme": "vancluever/acme@~> 2.10",
+  "ad": "ad@~> 0.4",
   "archive": "hashicorp/archive@~> 2.2",
   "aws": "aws@~> 4.0",
+  "awscc": "awscc@~> 0.33",
```

Comment:
@ansgarm Can you check in with Kit to see if this is worth adding? My understanding is they want to fold this into the main AWS provider, so if they're going to deprecate this within the next ~year, I don't know if it's worth us adding a prebuilt provider for it.